avconv.js

```diff
@@ -160,7 +221,7 @@ module.exports = function avconv(params) {
 
     avconv.on(eventType, function(exitCode, signal) {
         stream.end();
-        stream.emit('exit', exitCode, signal);
+        stream.emit('exit', exitCode, signal, parseMetaData(output));
```

Comment:
Well, we could reintroduce the `meta` event and use `stream.listeners('meta')` to check if there are listeners for it, but compared to the time avconv takes to do its job optimizing this seems unnecessary.